File: array/bubbleSort.cpp
/*  Bubble Sort:

*/

#include <iostream>
#include <limits.h>
using namespace std;

int main(){

    int arr[]= {2,7,4,9,1};
    int size = 5;

    // Print entered array
    cout << "Unsorted array: ";
    for (int i = 0; i < size; i++)
    {
        cout << arr[i] << " ";
    }
    cout << endl;

    for (int i = 0; i < size-1; i++)
    {
        for (int j = i+1; j < size; j++)
        {
            if (arr[j] < arr[i])
            {
                swap(arr[i], arr[j]);
            }            
        }        
    }

// Print entered array
    cout << "Sorted array: ";
    for (int i = 0; i < size; i++)
    {
        cout << arr[i] << " ";
    }
    

    return 0;
}

File: LeetCode/Sorting/Basic/BubbleSort/bubbleSort.js
// Bubble Sort
const arr = [2, 7, 9, 1, 45, 0, 67];

for (let i = 0; i < arr.length; i++) {
  for (let j = 0; j < arr.length - 1 - i; j++) {
    if (arr[j] > arr[j + 1]) {
      // Swap elements if they are out of order
      [arr[j], arr[j + 1]] = [arr[j + 1], arr[j]];
    }
  }
}

console.log(arr)